customers.go

```diff
@@ -260,3 +261,25 @@ func (api API) CreateCustomersContact(newContact *NewContact, customerUUID strin
 	path := strings.Replace(customerContactsEndpoint, ":uuid", customerUUID, 1)
 	return result, api.create(path, newContact, result)
 }
+
+// ListCustomerNotes
+//
+// See https://dev.chartmogul.com/reference/list-customer-notes
+func (api API) ListCustomerNotes(listCustomerNotesParams *ListNotesParams, customerUUID string) (*Notes, error) {
+	result := &Notes{}
+	query := make([]interface{}, 0, 1)
+	if listCustomerNotesParams != nil {
+		query = append(query, *listCustomerNotesParams)
+	}
+	path := customerNotesEndpoint + "?customer_uuid=" + customerUUID
```

Comment:
Why don't you append  `customer_uuid` into the `query`  instead? They're both query params